src/fetch.js

```diff
@@ -1,49 +1,6 @@
-import unfetch from 'isomorphic-unfetch'
-
-const RSF_VERSION_HEADER = 'x-rsf-api-version'
-const apiVersion = process.env.RSF_API_VERSION || '1'
+let fetch = require('isomorphic-unfetch')
 
 /**
- * An isomorphic implementation of the fetch API that always sends the x-rsf-api-version header.  You should
- * always use this to fetch data on both the client and server to ensure that React Storefront's
- * service worker serves responses cached by the service worker.
- *
- * @param {String} url URL to `fetch`.
- * @param {Object} opts Options passed to the `fetch` call.
- * @return {Promise} The `Promise` returned from the `fetch` call.
+ * An isomorphic implementation of the fetch API. You should always use this to fetch data on both the client and server.
  */
-export default function fetch(url, opts = {}) {
-  const headers = opts.headers || {}
-  headers[RSF_VERSION_HEADER] = apiVersion
-  return unfetch(url, { ...opts, headers })
-}
-
-/**
- * @private
- * Adds x-rsf-api-version to all xhr
- */
-function monkeyPatchXHR() {
-  const open = XMLHttpRequest.prototype.open
-
-  XMLHttpRequest.prototype.open = function() {
-    const res = open.apply(this, arguments)
-    this.setRequestHeader(RSF_VERSION_HEADER, apiVersion)
```

Comment:
Is this header not needed anymore?

Reply:
That's correct, we've switched to adding the next build id to the URL, so workbox can easily cache by path, rather than needing custom behavior.